processor/spanmetricsprocessor/internal/cache/cache.go

```diff
@@ -52,17 +54,27 @@ func (c *Cache[K, V]) RemoveEvictedItems() {
 	}
 }
 
-// Get retrieves an item from the LRU cache or evicted items.
+// Add a value to the cache, returns true if an eviction occurred and updates the "recently used"-ness of the key.
+func (c *Cache[K, V]) Add(key K, value V) bool {
```

Comment:
Why is this func added? Doesn't seem to be used

Reply:
It is, before was exposed because of embedding public `lru.Cache`, now that it is a private variable no longer exposed.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/spanmetricsprocessor/processor.go#L503

Reply:
Got it. thanks for explaining!